### DISCUSSION TITLE
旁路由下，iStoreOS 里面的 Docker  插件里面没有网络就解决办法

### DISCUSSION THREAD

**Discussion by ks-bao** [istoreos/istoreos#1077] [General]
1. 打开网络，防火墙，区域，LAN 里面的 IP 动态 伪装打勾

![image](https://github.com/istoreos/istoreos/assets/88376714/fd7f33a4-4ce7-4fa1-9948-f8342f88f572)

2. NAT 规则里面，新建一个专门针对 Docker 接口的规则

![image](https://github.com/istoreos/istoreos/assets/88376714/1a9c827c-818f-4409-9cdd-02e54396887d)

![image](https://github.com/istoreos/istoreos/assets/88376714/759b0bb6-4167-4b02-a4e8-60e5a4bf1589)

**Comment by jjm2473** [comment]
新固件已经默认打开docker的nat了
